pkg/authentication/authentication.go

```diff
@@ -19,6 +19,11 @@ import (
 	"github.com/superplanehq/superplane/pkg/models"
 )
 
+const (
+	ProviderGitHub = "github"
```

Comment:
Let's keep these constants in the `pkg/models` package. While working on the task to allow the secrets/integrations on the organization level too, I've ran into issues with import cycles by having the domain type scope (org/canvas) in the pkg/authorization